CIScripts/Test/PesterLogger/RemoteLogCollector.Tests.ps1

```diff
@@ -211,10 +212,32 @@ Describe "RemoteLogCollector - with actual Testbeds" -Tags CI, Systest {
 
     AfterEach {
         Remove-Item "TestDrive:/*" 
-        if (Get-Item function:Write-Log -ErrorAction SilentlyContinue) {
-            Remove-Item function:Write-Log
+        if (Get-Item function:Write-LogImpl -ErrorAction SilentlyContinue) {
+            Remove-Item function:Write-LogImpl
         }
     }
 
     Test-MultipleSourcesAndSessions
+
+    Context "Docker logs" {
+        BeforeEach {
+            Initialize-PesterLogger -OutDir "TestDrive:\"
+        }
+
+        It "captures logs of container" {
+            New-Container -Session $Sess1 -Name foo -Network nat
+
+            Merge-Logs (New-ContainerLogSource -Session $Sess1 -Container foo)
+            $ContentRaw = Get-Content -Raw "TestDrive:\*.Docker logs.captures logs of container.log"
+            $ContentRaw | Should -BeLike "*Microsoft Windows*"
```

Comment:
Could you provide an example log with two types of log sources?

Reply:
Do you mean a test?